src/app/workflow/workflow.component.html

```diff
@@ -449,6 +480,24 @@ <h3>
     </div>
     <div fxLayout="column" fxLayoutGap="1rem" class="p-0 col-sm-12 col-md-3 col-lg-3 ng-scope" *ngIf="isWorkflowPublic && workflow">
       <app-launch-third-party [workflow]="workflow" [selectedVersion]="selectedVersion"></app-launch-third-party>
+      <mat-card *ngIf="workflow.conceptDois[workflow.doiSelection]">
+        <div class="h5">Cite this workflow</div>
```

Comment:
Should you be using `workflow.entryTypeMetadata.term` here? Is it possible it could be a notebook here?